examples/eks-cluster-with-observability/helm_values/aws-for-fluentbit-values.yaml

```diff
@@ -0,0 +1,19 @@
+serviceAccount:
```

Comment:
If this file is no different than the default values.yaml in the addon then no need to pass it.

Reply:
I think he needs this config with elasticsearch enabled